package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-loop",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
can you remove the version bump please? i can bump and tag master after this merged.